openmc/opencg_compatible.py

```diff
@@ -196,59 +196,59 @@ def get_opencg_surface(openmc_surface):
         raise ValueError(msg)
 
     global OPENCG_SURFACES
-    surface_id = openmc_surface._id
+    surface_id = openmc_surface.id
 
     # If this Material was already created, use it
     if surface_id in OPENCG_SURFACES:
         return OPENCG_SURFACES[surface_id]
 
     # Create an OpenCG Surface to represent this OpenMC Surface
-    name = openmc_surface._name
+    name = openmc_surface.name
 
     # Correct for OpenMC's syntax for Surfaces dividing Cells
-    boundary = openmc_surface._boundary_type
+    boundary = openmc_surface.boundary_type
     if boundary == 'transmission':
         boundary = 'interface'
 
     opencg_surface = None
 
-    if openmc_surface._type == 'plane':
-        A = openmc_surface._coeffs['A']
-        B = openmc_surface._coeffs['B']
-        C = openmc_surface._coeffs['C']
-        D = openmc_surface._coeffs['D']
+    if openmc_surface.type == 'plane':
+        A = openmc_surface.coeffs['A']
+        B = openmc_surface.coeffs['B']
+        C = openmc_surface.coeffs['C']
+        D = openmc_surface.coeffs['D']
```

Comment:
Any reason you don't use the `a`, `b`, `c`, and `d` properties directly? Same comment goes for all the other surfaces.